modules/apps/commerce/commerce-product-content-search-web/src/main/java/com/liferay/commerce/product/content/search/web/internal/display/context/CPSearchResultsDisplayContext.java

```diff
@@ -231,12 +233,20 @@ public String getOrderByCol() {
 		String portletId = ParamUtil.getString(
 			originalHttpServletRequest, "p_p_id");
 
-		return ParamUtil.getString(
+		String orderByCol = ParamUtil.getString(
 			originalHttpServletRequest,
 			StringBundler.concat(
 				StringPool.UNDERLINE, portletId, StringPool.UNDERLINE,
 				SearchContainer.DEFAULT_ORDER_BY_COL_PARAM),
-			"relevance");
+			CPSearchResultsConstants.DEFAULT_SORT_OPTION);
+
+		if (ArrayUtil.contains(
```

Comment:
what if someone put something like:
price-low-to-high<script>alert(123)</script>?

Reply:
It doesn't cause problem because it's not one of the allowed values of the constant SORT_OPTIONS.